mojave/items/medical/stack_medical.dm

```diff
@@ -0,0 +1,73 @@
+//Children of base TG stack medical items should go here
+
+/obj/item/stack/medical/suture/ms13
+    name = "doctor's bag"
+    singular_name = "use" //This is so that examine text says "X uses left in the stack" instead of "X doctor's bags left in the stack"
+    desc = "A bag filled with medical tools and supplies for patching and stabilizing brute damage and bleeding."
+    w_class = WEIGHT_CLASS_SMALL
+    full_w_class = WEIGHT_CLASS_SMALL
+    max_amount = 12
+    amount = 12
+    self_delay = 35
+    other_delay = 20
+    gender = NEUTER //So examine text says "This is a doctor's bag" instead of "These are some doctor's bag"
```

Comment:
```suggestion
    gender = NEUTER //So examine text says "This is a doctor's bag" instead of "These are some doctor's bag"
```
But why? It's **a** doctor's bag, not several doctor's bags.